cinema/urls.py

```diff
@@ -15,13 +15,20 @@
 """
 from django.contrib import admin
 from django.urls import path, include
+from django.conf import settings
+from django.conf.urls.static import static
+from Schedule_app.views import CinemaList, HallList
 
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('', include('basepage.urls')),
     path('users/', include('django.contrib.auth.urls')),
     path('users/', include('app_users.urls')),
     # path('schedule_movie/', include('Schedule_app.urls'))
-    path('book_ticket/', include('Schedule_app.urls'))
+    path('book_ticket/', include('Schedule_app.urls')),
+    path('ajax/cinema/', CinemaList.as_view()),
+    path('ajax/hall/', HallList.as_view()),
 
 ]
+urlpatterns += static(settings.STATIC_URL, document_root=settings.STATIC_ROOT)
+urlpatterns += static(settings.MEDIA_URL, document_root=settings.MEDIA_ROOT)
```

Comment:
Normally, you would want to do this only in the case of the local development environment. In production, these files would normally be served by a web server like Apache or Nginx.